nav2_behavior_tree/plugins/decorator/speed_controller.cpp

```diff
@@ -30,19 +30,28 @@ SpeedController::SpeedController(
   first_time_(false),
   period_(1.0)
 {
+  node_ = config().blackboard->get<rclcpp::Node::SharedPtr>("node");
+
   getInput("min_rate", min_rate_);
   getInput("max_rate", max_rate_);
   getInput("min_speed", min_speed_);
   getInput("max_speed", max_speed_);
 
+  if (min_rate_ <= 0) {
+    RCLCPP_WARN(
+      node_->get_logger(),
+      "min_rate cannot be <= 0, setting to a default value of 0.01 hz");
+    min_rate_ = 0.01;
```

Comment:
I think this should be a log fatal and then throw an exception. This is an impossible configuration.